src/lib/components/MarkdownBody.svelte

```diff
@@ -3,17 +3,28 @@
   import { marked } from "marked";
   import { twMerge } from "tailwind-merge";
 
+  export let bare: boolean = false;
+  export let wrappedLink: boolean = false; // if body is wrapped in an anchor tag (has to be positioned absolute and inset-0)
   let clazz: string = "";
   export { clazz as class };
   export let body: string;
+  $: mergedClass = (() => {
+    if (bare) return clazz;
+    let merged = twMerge(
+      "prose prose-a:link-primary prose-a:no-underline",
+      clazz,
+    );
+    if (wrappedLink) {
+      merged = twMerge(
+        "pointer-events-none relative prose-a:pointer-events-auto",
+        merged,
+      );
+    }
+    return merged;
+  })();
```

Comment:
Instead of doing this, could we not just avoid using anchor tags on the landing page?
```
const renderer = {
    link(href: string, title: string | null | undefined, text: string) {
      return text;
    },
  };
const customMarked = wrappedLink ? new Marked({ renderer }) : new Marked();
```

and using `customMarked.parse` instead of `marked` on line 30.

I feel like this is both easier to understand and takes less lines of code. This would also solve the problem of being able to tab to links in parts of the text that shouldn't be visible.

Reply:
I don't really think that the links needs to be clickable here. Usually you would click on the article to read the whole thing before clicking any links. I also disagree that all links in the text should be tabbable here, it looks weird if the text scrolls when tabbing, and there is no other way to scroll the text other than tabbing to links.

Reply:
Fine I can agree with this I guess. I did not understand you meant link not visible, they should of course not be tabbable to. I just meant the links actually visible within the line clamp. But I can settle for having to first go to the article and then press the links.